app/controllers/turns_controller.rb

```diff
@@ -0,0 +1,10 @@
+class TurnsController < ApplicationController
+
+  def index
+    @turn = Turn.all
+  end
+
+  def show; end
+
+  def create; end
+end
```

Comment:
Please add a new line at the end of the file

Reply:
Ok, I'm going to add a new line 🧐